### PR TITLE
ACMS-1484: Content preview headless test case.

### PR DESCRIPTION
**Pull request by apathak18**
**Motivation**

Fixes ACMs-1484

**Proposed changes**
Content preview test case

### CODE REVIEW

File: modules/acquia_cms_headless/tests/src/Functional/SitePreviewTest.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+/**
+ * Tests headless content site preview.
+ *
+ * @group acquia_cms
+ * @group acquia_cms_headless
+ * @group medium_risk
+ * @group push
+ */
+class SitePreviewTest extends WebDriverTestBase {
```
Comment (deepakmishra2) [inline modules/acquia_cms_headless/tests/src/Functional/SitePreviewTest.php:17]: Please use HeadlessTestBase.

@apathak18

**Review by rajeshreeputra — APPROVED** [review]
LGTM

**Review by chandan-singh7929 — APPROVED** [review]
LGTM

**Comment by rajeshreeputra** [comment]
please rebase the branch.

**Review by vishalkhode1 — APPROVED** [review]

File: modules/acquia_cms_headless/tests/src/Functional/SitePreviewTest.php (hunk 1 above)
Comment (apathak18) [reply modules/acquia_cms_headless/tests/src/Functional/SitePreviewTest.php:16]: @deepakmishra2 Not required as I have to login via admin and created a test content type in setup.

**Pull request merged**